.github/workflows/audit.yml

```diff
@@ -13,6 +14,10 @@ jobs:
     runs-on: ubuntu-latest
```

Comment:
It doesn't let me make suggestions on the line above this one, but can we change the job name to `audit` or split it into `audit-rust` and `audit-js`?